src/main/kotlin/pt/up/fe/ni/website/backend/controller/ProjectController.kt

```diff
@@ -65,25 +65,25 @@ class ProjectController(private val service: ProjectService) {
     @PutMapping("/{id}/unarchive")
     fun unarchiveProjectById(@PathVariable id: Long) = service.unarchiveProjectById(id)
 
-    @PutMapping("/{idProject}/addTeamMember/{idAccount}")
+    @PutMapping("/{idProject}/team/{idAccount}")
     fun addTeamMemberById(
         @PathVariable idProject: Long,
         @PathVariable idAccount: Long
     ) = service.addTeamMemberById(idProject, idAccount)
 
-    @PutMapping("/{idProject}/removeTeamMember/{idAccount}")
+    @DeleteMapping("/{idProject}/team/{idAccount}")
     fun removeTeamMemberById(
         @PathVariable idProject: Long,
         @PathVariable idAccount: Long
     ) = service.removeTeamMemberById(idProject, idAccount)
 
-    @PutMapping("/{idProject}/addHallOfFameMember/{idAccount}")
+    @PutMapping("/{idProject}/hallOfFameMember/{idAccount}")
```

Comment:
```suggestion
    @PutMapping("/{idProject}/hallOfFame/{idAccount}")
```
I think this is more similar to "team", wdyt?

Reply:
Good catch. Already updated it, take a look.